test/integration/convert_partial_specialization_test.cpp

```diff
@@ -0,0 +1,89 @@
+#include "yaml-cpp/emitterstyle.h"
+#include "yaml-cpp/eventhandler.h"
+#include "yaml-cpp/yaml.h"  // IWYU pragma: keep
+#include "gtest/gtest.h"
+
+// Base class
+class A {
+ public:
+  // virtual load/emit methods
+  virtual void load(const YAML::Node &node) { a = node["a"].as<int>(); }
+
+  virtual YAML::Node emit() const {
+    YAML::Node node;
+    node["a"] = a;
+    return node;
+  }
+
+  int a;
+};
+
+// Derived class
+class B : public A {
+ public:
+  // override virtual load/emit methods
+  virtual void load(const YAML::Node &node) override {
```

Comment:
Even though it's a test case, I think the constructors are needed, like #310. It is also convenient for later usage.

Reply:
Added

Reply:
How about updating the Tutorial.md at the same time. And use this in testcase.
```C++  
  A a(1);
  B b(1, 2);
```

Reply:
I updated the test cases to use the constructors and added them in the tutorial.
However the tutorial doesn't make use of them